lib/realtime/data_status.ex

```diff
@@ -1,8 +1,51 @@
 defmodule Realtime.DataStatus do
+  alias Realtime.Vehicle
+
   @type t :: :good | :outage
 
-  @spec data_status() :: t()
-  def data_status() do
-    :good
+  @spec data_status([Vehicle.t()]) :: t()
+  def data_status(vehicles) do
+    now_fn = Application.get_env(:skate, :now_fn, &Util.Time.now/0)
+    now = now_fn.()
+
+    {good_count, bad_count} =
+      vehicles
+      |> Enum.filter(&should_consider?/1)
+      |> count_partition(fn vehicle -> vehicle_is_good(vehicle, now) end)
+
+    total_count = good_count + bad_count
+
+    if bad_count >= 2 and bad_count / total_count > 0.20 do
```

Comment:
What happens if all the vehicles are stale?

Reply:
If all of them are unassigned, then we'll show the status as good (but all the buses will be ghosts, so the problem will be obvious).

If all assigned buses are stale, then we'll show the outage banner.